# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/snes9x/hotfixes.patch
# Copyright (C) 2023 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- snes9x/vulkan/slang_shader.hpp.vanilla	2023-10-20 22:36:25.904721328 +0200
+++ snes9x/vulkan/slang_shader.hpp	2023-10-20 22:36:47.853722147 +0200
@@ -1,4 +1,6 @@
 #pragma once
+
+#include <stdint.h>
 #include <string>
 #include <vector>
 
@@ -101,4 +103,4 @@
     int ubo_binding;
     std::vector<Uniform> uniforms;
     std::vector<Sampler> samplers;
-};
\ No newline at end of file
+};
--- snes9x/external/VulkanMemoryAllocator-Hpp/include/vk_mem_alloc.h.vanilla	2023-10-20 22:37:54.388724628 +0200
+++ snes9x/external/VulkanMemoryAllocator-Hpp/include/vk_mem_alloc.h	2023-10-20 22:38:20.790725613 +0200
@@ -121,6 +121,7 @@
 See documentation chapter: \ref statistics.
 */
 
+#include <stdio.h>
 
 #ifdef __cplusplus
 extern "C" {
